services/collaboration/pkg/connector/fileconnector.go

```diff
@@ -347,10 +368,27 @@ func (f *FileConnector) RefreshLock(ctx context.Context, lockID string) (*Connec
 		return nil, err
 	}
 
+	statResp, err := f.gwc.Stat(ctx, &providerv1beta1.StatRequest{
```

Comment:
hm, whenever we make a CS3 call we need to select the next client. this will always reuse the same client and I fear this will cause disconnects when scaling in kubernetes.

Reply:
Yes, that would affect all cs3 calls.